pkg/destroy/aws/ec2helpers.go

```diff
@@ -786,7 +786,38 @@ func deleteEC2VPCPeeringConnection(ctx context.Context, client *ec2.EC2, id stri
 }
 
 func deleteEC2VPCEndpointService(ctx context.Context, client *ec2.EC2, id string, logger logrus.FieldLogger) error {
-	_, err := client.DeleteVpcEndpointServiceConfigurationsWithContext(ctx, &ec2.DeleteVpcEndpointServiceConfigurationsInput{
+	output, err := client.DescribeVpcEndpointConnectionsWithContext(ctx, &ec2.DescribeVpcEndpointConnectionsInput{
+		Filters: []*ec2.Filter{
+			{
+				Name:   aws.String("service-id"),
+				Values: aws.StringSlice([]string{id}),
+			},
+		},
+	})
+
+	if err != nil {
+		logrus.Warn("Unable to get the list of VPC endpoint connections connected to service: ", err)
+		logrus.Warn("Attempting to delete the VPC Endpoint Service")
```

Comment:
So any specific reason for not using `logger` passed in as argument? That way we could track this error back to the specific resource `id` that caused it.

Reply:
Was putting more focus on if this actually works. Fixed it.